components/builder-web/app/util.ts

```diff
@@ -22,7 +22,7 @@ import { FeatureFlags } from './privilege';
 export function createGitHubLoginUrl(state) {
   const params = {
     client_id: config['github_client_id'],
-    redirect_uri: `${window.location.protocol}//${window.location.host}/`,
+    redirect_uri: `${config['github_redirect_uri']}`,
```

Comment:
Let's have this fall back to the current URI, as it does today, in case the config value isn't present (which it won't be when we merge this), since we know that currently works across the board. Something like this:

```
redirect_uri: `${config['github_redirect_uri'] || (window.location.protocol + '//' + window.location.host + '/')}`,
```
... ought to do it.